src/ConnectionRegistry.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Doctrine\Bundle\DBALBundle;
+
+use Doctrine\DBAL\Connection;
+
+interface ConnectionRegistry
```

Comment:
we should probably extend `Doctrine\Common\Persistence\ConnectionRegistry`?

Reply:
well, we cannot if we want to have types (we would lie regarding variance of return types). Thus, the plan would be to deprecate methods of `Doctrine\Persistence\ConnectionRegistry` (and ultimately drop this interface), as it brings no benefit (there is no cross-project concept of connection in Doctrine. The DBAL and Mongo connections have no API in common).